transactionaldriver-and-tomcat/run.sh

```diff
@@ -0,0 +1,21 @@
+wget -nc http://mirror.vorboss.net/apache/tomcat/tomcat-7/v7.0.78/bin/apache-tomcat-7.0.78.zip
```

Comment:
IMO it's tomcat over here: 
https://github.com/tomjenkinson/quickstart/blob/JBTM-2766/transactionaldriver-and-tomcat/src/main/webapp/META-INF/context.xml#L14